src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -247,19 +248,19 @@ after_frag( void *             _ctx,
       FD_LOG_ERR(( "unexpected fork switch mid block execution, cannot continue" ));
     }
 
-    int res = fd_runtime_prepare_txns( &fork->slot_ctx, task_info, txns, txn_cnt );
-    if( res != 0 && !( ctx->flags & REPLAY_FLAG_PACKED_MICROBLOCK ) ) {
-      FD_LOG_WARNING(( "block invalid - slot: %lu", ctx->curr_slot ));
-      // TODO: need to yeet this block from the replay frontier.
-      return;
-    }
+    // int res = fd_runtime_prepare_txns( &fork->slot_ctx, task_info, txns, txn_cnt );
```

Comment:
Why is this commented out?

Reply:
it was replaced with `fd_runtime_execute_txns_in_waves_tpool`